CODIGO/General.bas

```diff
@@ -1065,6 +1065,9 @@ On Error GoTo Main_Err
     Call new_engine_init(ao20rendering.renderer)
 #Else
     Call engine_init 'initializes DX
+    Debug.Assert Not DirectX Is Nothing
```

Comment:
in prev version we would check if sound is disabled before trying to initialize the audio engine, this is useful since some users have issues with the game starting if they don't have any audio device connected to the computer

Reply:
Right, correct, but this check has nothing to do with sound. It's just an assert making sure the DirectX8 object was created without problems.

The behavior with the new audio system is the same as before and the game won't try to initialize audio when the variable `ao20audio.AudioEnabled=0`

So not a problem here really